pandoc.cabal

```diff
@@ -180,6 +181,7 @@ Extra-Source-Files:
                  test/writer.markdown
                  test/writer.plain
                  test/writer.mediawiki
+                 test/writer.muse
```

Comment:
I think you need to add `test/writer.muse` to the repository -- I don't see it here in this PR.